spec/controllers/api/v2/searches_controller_spec.rb

```diff
@@ -512,6 +493,7 @@
       before do
         affiliate = mock_model(Affiliate, api_access_key: 'usagov_key', locale: :en, search_engine: 'BingV6')
         expect(Affiliate).to receive(:find_by_name).and_return(affiliate)
+        allow(affiliate).to receive_message_chain(:routed_queries, :joins, :where, :first)
```

Comment:
Stub the `search_engine` on the fixture affiliate, instead of the message chains.